en/faq.md

```diff
@@ -0,0 +1,77 @@
+
+The limit is in theory 255 vehicles for system IDs ranging from 1 to 255.
+
+### Is MAVLink 1 supported?
+
+No. MAVSDK only supports [MAVLink 2.0](https://mavlink.io/en/guide/mavlink_2.html).
+
+### What sorts of vehicles are supported?
+
+The MAVSDK API is designed for interacting with *aircraft*. It has primarily been tested for use with multicopters, but also has basic support for fixed wing and [VTOL](../guide/vtol.md).
+
+The API *may* "work" with ground-based or other types of vehicles, but some methods will not make sense.
+This use-case is mostly unsupported and untested.
+
+### Does MAVSDK support indoor use?
+
+Indoor use is supported, however, some modes such as mission or position control are not available indoor, unless some additional positioning method is available (e.g. optical flow, visual-inertial odometry, a motion capture system, etc.).
+
+Note that PX4 currently does not support missions in "local coordinates" i.e. meters but only "global coordinates i. e. latitude/longitude.
+
+### What UAV flight stacks are supported?
+
+MAVSDK, so far, is optimised for use with the PX4 flight stack and all testing is done against PX4.
+
+While many features should work on other flight stacks there may be implementation differences at the [MAVLink microservices level](https://mavlink.io/en/protocol/overview.html) that mean not every API will work.
+For example, PX4 and ArduPilot implement the parameter protocol differently, and vary slightly in the mission upload/download protocol (e.g. ArduPilot uses the 0 entry as the home position).
+
+> **Note** The SDK welcomes contributions to better support flight stacks other than PX4.
+> We do however expect contributors to also help with testing and maintenance support for using the SDK with their autopilot.
+
+### Are serial connections supported?
+
+Yes. Serial, TCP, and UDP connections are supported.
+
+### Why is libCURL a dependency?
+
+libCURL will be required to download the camera definition file which is referenced in [CAMERA_INFORMATION](https://mavlink.io/en/messages/common.html#CAMERA_INFORMATION).
+It might also come in handy whenever any other REST resources are required.
+
+### How are plugins added?
+
+Check out [contributing/plugins](../contributing/plugins.md).
+
+### Can a plugin depend on another one?
+
+Yes - but it should not!
+The idea is that plugins don't have any coupling between each other which allows for any plugin to be changed or removed without breaking the rest.
+This will lead to some duplication at places, that's an acceptable trade-off.
+If the same code is used in many places, it can always be moved to core and get re-used from there.
+
+Over time we have sometimes moved functionality from plugins to the core if it was exposed in multiple plugins (e.g. mission upload/download has been moved to the core so it can be used in the Mission plugin as well as the MissionRaw and Geofence plugins.
+
+### What is the difference between unit and integration tests?
```

Comment:
This feels like it is too detailed for here. I would remove. Or at least change to "How robust is the software" and explain that we have testing (with link to info)